CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.10.2
+
+<!-- generated comparing v5.10.1..master -->
+
+_Aug 22, 2022_
+
+A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ @michaldudak synced the Material Icons set with the latest from Google (#33988).  
+  A couple of icons changed their appearance. See the difference [on this Argos build](https://app.argos-ci.com/mui/material-ui/builds/4428]).
```

Comment:
I have fixed https://github.com/mui/material-ui/releases/tag/v5.10.2, the link was not working.

```suggestion
  A couple of icons changed their appearance. See the difference [on this Argos build](https://app.argos-ci.com/mui/material-ui/builds/4428).
```